diff --git a/src/macappkit.m b/src/macappkit.m
index 825b4e279e..d3c322fdcd 100644
--- a/src/macappkit.m
+++ b/src/macappkit.m
@@ -2763,10 +2772,8 @@ - (void)setupWindow
       if (!self.shouldBeTitled)
 	windowStyle = NSWindowStyleMaskBorderless;
       else
-	windowStyle = (NSWindowStyleMaskTitled | NSWindowStyleMaskClosable
-		       | NSWindowStyleMaskMiniaturizable
-		       | NSWindowStyleMaskResizable);
-    }
+	windowStyle = (NSWindowStyleMaskBorderless | NSWindowStyleMaskResizable);
+    }
   else
     windowStyle = NSWindowStyleMaskBorderless;
 
