diff --git a/src/bar.c b/src/bar.c
index 4f502d4..7ca2648 100644
--- a/src/bar.c
+++ b/src/bar.c
@@ -244,16 +244,10 @@ void bar_refresh(struct bar *bar)
                 pos.x += 25;
             }

-            bar_draw_line(bar, space_line, pos.x, pos.y);
-
-            if (sid == space_list[i]) {
-                CGPoint new_pos = CGContextGetTextPosition(bar->context);
-                struct bar_line mark_line = g_bar_manager.space_underline;
-                CGPoint mark_pos = bar_align_line(bar, mark_line, 0, ALIGN_BOTTOM);
-                mark_pos.x = mark_pos.x - mark_line.bounds.size.width / 2 - space_line.bounds.size.width / 2;
-                bar_draw_line(bar, mark_line, mark_pos.x, mark_pos.y);
-                CGContextSetTextPosition(bar->context, new_pos.x, new_pos.y);
-            }
+	    if (sid == space_list[i]) {
+	      space_line.color = rgba_color_from_hex(0xffd75f5f); // override color used when drawing the active space
+	    }
+	    bar_draw_line(bar, space_line, pos.x, pos.y);

-            final_bar_left_x = pos.x + space_line.bounds.size.width + 10;
-        }
+	    final_bar_left_x = pos.x + space_line.bounds.size.width + 10;
+	}

         free(space_list);
     }
